app/javascript/mastodon/containers/compose_container.js

```diff
@@ -29,7 +30,9 @@ export default class TimelineContainer extends React.PureComponent {
     return (
       <IntlProvider locale={locale} messages={messages}>
         <Provider store={store}>
-          <Compose />
+          <BrowserRouter basename='/web' forceRefresh>
+            <Compose />
+          </BrowserRouter>
```

Comment:
What is this for?

Reply:
For `Link` of React Router DOM. It does no routing, but it just navigates to the linked page.